### PR TITLE
feat: Adds timestamp equality steps

### PR DESCRIPTION
**Pull request by x2012x**
Adds two equality steps for timestamps.

### CODE REVIEW

File: lib/grpc-api-assistant/grpc_steps.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'time'
```
Comment (michaelkeeling) [inline lib/grpc-api-assistant/grpc_steps.rb:1]: [I've been trying to keep all the requires statements in a single place](https://github.com/michaelkeeling/cucumber-grpc-api-assistant/blob/master/lib/grpc-api-assistant/env.rb).

**Review by michaelkeeling — APPROVED** [review]
Good stuff.  Just move the `require` statement and I think this will be good to go if the tests all pass.

File: lib/grpc-api-assistant/grpc_steps.rb (hunk 1 above)
Comment (x2012x) [reply lib/grpc-api-assistant/grpc_steps.rb:1]: Moving the require statement to that file doesn't work. But, if it's moved to [this](https://github.com/michaelkeeling/cucumber-grpc-api-assistant/blob/master/lib/grpc-api-assistant.rb#L13-L14) file, it does. What do ya think?

Comment (michaelkeeling) [reply lib/grpc-api-assistant/grpc_steps.rb:1]: it looks like the autoloading in either Ruby or Cucumber (or both) changed at some point with the upgrades I just pushed up.  Don't worry about this change.  I'm not 100% what that `.env` file is doing now 😛

**Review by michaelkeeling — APPROVED** [review]

**Pull request merged**